ui/lib/kv/addon/templates/list-directory.hbs

```diff
@@ -0,0 +1,10 @@
+<Page::List
+  @secrets={{this.model.secrets}}
+  @backend={{this.model.backend}}
+  @pathToSecret={{this.model.pathToSecret}}
+  @pageFilter={{this.model.pageFilter}}
+  @filterValue={{this.model.filterValue}}
+  @noMetadataListPermissions={{this.model.noMetadataListPermissions}}
+  @breadcrumbs={{this.breadcrumbs}}
+  @routeName={{this.routeName}}
```

Comment:
I removed this arg in another PR https://github.com/hashicorp/vault/pull/22488/files#diff-0582cc457b95f076c227bc7a64f531c93d4a1c2e71dd385417e35cee7e042be0